intfsyncd/intfsync.cpp

```diff
@@ -47,6 +47,11 @@ void IntfSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
 
     key = LinkCache::getInstance().ifindexToName(rtnl_addr_get_ifindex(addr));
+
+    /* Don't sync lo, eth0, and docker0 routes */
+    if (key == "lo" || key == "eth0" || key == "docker0")
+        return;
```

Comment:
these are hardcoded values. Can you check if the key in the your front panel port list? That's more generic. as in some platform they may have other interfaces such as eth1, ... etc.